boards/sodaq-sara-aff/Kconfig

```
@@ -0,0 +1,16 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config BOARD
+    default "sodaq-sara-aff" if BOARD_SODAQ_SARA_AFF
+
+config BOARD_SODAQ_SARA_AFF
+    bool
+    default y
+    select CPU_MODEL_SAMD21J18A
+    select BOARD_COMMON_SODAQ
```

In 05a5f3cedae96aa3b937bf7c499d9d71c76febe2 you moved the common symbol at the first place, for consistency, this should be the same here and with other sodaq boards.

```suggestion
    select BOARD_COMMON_SODAQ
    select CPU_MODEL_SAMD21J18A
```